pkg/plugins/builder.go

```diff
@@ -79,26 +79,26 @@ func NewBuilder(client *Client, plugins map[string]Descriptor, localPlugins map[
 
 		logger := log.With().Str("plugin", "plugin-"+pName).Str("module", desc.ModuleName).Logger()
 
-		i, wasmPath, err := initMiddlewareBuilder(logger, localGoPath, desc.ModuleName, manifest)
-		if err != nil {
-			return nil, fmt.Errorf("%s: %w", desc.ModuleName, err)
-		}
-
 		switch manifest.Type {
 		case middleware:
 			switch manifest.Runtime {
 			case RuntimeWasm:
-				pb.wasmMiddlewareBuilders[pName] = newWasmMiddlewareBuilder(wasmPath)
+				pb.middlewareBuilders[pName] = newWasmMiddlewareBuilder(client.GoPath(), desc.ModuleName, manifest)
```

Comment:
hmm instead of client.GoPath() I was using `localGoPath` in case of local plugins.